framework/src/main/java/org/tron/core/services/http/solidity/SolidityNodeHttpApiService.java

```diff
@@ -184,9 +184,11 @@ public void start() {
       context.addServlet(new ServletHolder(triggerConstantContractServlet),
           "/walletsolidity/triggerconstantcontract");
 
-      context.addServlet(new ServletHolder(getNodeInfoServlet), "/wallet/getnodeinfo");
+      context.addServlet(new ServletHolder(getNodeInfoServlet), "/walletsolidity/getnodeinfo");
```

Comment:
Whether it is necessary to change ‘wallet’ to ‘walletsolidity’ needs to be considered again